Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2005-08-18
Initial Package Version: 1.2.1
Upstream Status:         Not submitted (idea for the patch was discovered from a
                         mailing list message on the subversion mailing list)
Origin:                  Max Bowsher
                         http://svn.haxx.se/users/archive-2005-07/0136.shtml
Description:             Fixes build issue using swig-1.3.25

$LastChangedBy$
$Date$


diff -Naur subversion-1.2.1-orig/subversion/bindings/swig/core.i subversion-1.2.1/subversion/bindings/swig/core.i
--- subversion-1.2.1-orig/subversion/bindings/swig/core.i	2005-07-01 17:25:14.000000000 +0000
+++ subversion-1.2.1/subversion/bindings/swig/core.i	2005-08-18 23:14:12.000000000 +0000
@@ -425,13 +425,13 @@
 
 static apr_pool_t *current_pool = 0;
 
-SWIGEXPORT(apr_pool_t *)
+SWIGEXPORT apr_pool_t *
 svn_swig_pl_get_current_pool (void)
 {
   return current_pool;
 }
 
-SWIGEXPORT(void)
+SWIGEXPORT void
 svn_swig_pl_set_current_pool (apr_pool_t *pool)
 {
   current_pool = pool;
